templates/web/eastherts/header.html

```diff
@@ -0,0 +1,66 @@
+[%
+    SET html_att = ' lang="' _ lang_code _ '"';
+    # For a right-to-left language, use the following line in your own header:
+    # SET html_att = html_att _ ' dir="rtl"';
+-%]
+<!doctype html>
+<!--[if lt IE 7]><html class="no-js ie6 iel8"[% html_att %]><![endif]-->
+<!--[if IE 7]>   <html class="no-js ie7 iel8"[% html_att %]><![endif]-->
+<!--[if IE 8]>   <html class="no-js ie8 iel8"[% html_att %]><![endif]-->
+<!--[if IE 9]>   <html class="no-js ie9"[% html_att %]><![endif]-->
+<!--[if gt IE 9]><!--><html class="no-js"[% html_att %]><!--<![endif]-->
+    <head>
+        <meta name="viewport" content="initial-scale=1.0">
+
+        <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
+        <meta name="HandHeldFriendly" content="true">
+        <meta name="mobileoptimized" content="0">
+
+        [% INCLUDE 'header_opengraph.html' %]
+        [% SET start = c.config.ADMIN_BASE_URL IF admin;
+
+           # For clarity, the 'fixmystreet' moniker (for fixmystreet.com) puts
+           # it stylesheets under fixmystreet.com
+           IF c.cobrand.moniker == 'fixmystreet';
+               SET css_dir = 'fixmystreet.com';
+           ELSE;
+               SET css_dir = c.cobrand.moniker;
+           END %]
+        <link rel="stylesheet" href="[% start %][% version('/cobrands/' _ css_dir _ '/base.css') %]">
+        <link rel="stylesheet" href="[% start %][% version('/cobrands/' _ css_dir _ '/layout.css') %]" media="(min-width:48em)">
+        [% extra_css %]
+        <!--[if (lt IE 9) & (!IEMobile)]>
+            <link rel="stylesheet" href="[% start %][% version('/cobrands/' _ css_dir _ '/layout.css') %]">
+        <![endif]-->
+
+        <script src="[% start %][% version('/js/modernizr.custom.js') %]" charset="utf-8"></script>
+        <script src="[% start %][% version('/cobrands/fixmystreet/position_map.js') %]" charset="utf-8"></script>
+        [% INCLUDE 'common_header_tags.html' %]
+        [% extra_js %]
+
+        [% IF c.req.uri.host == 'osm.fixmystreet.com' %]
+            <link rel="canonical" href="https://www.fixmystreet.com[% c.req.uri.path_query %]">
+        [% END %]
+
+        [% TRY %][% PROCESS 'header_extra.html' %][% CATCH file %][% END %]
+
+    </head>
+    [% TRY %][% PROCESS 'set_body_class.html' %][% CATCH file %][% END %]
+    <body class="[% bodyclass | html IF bodyclass %]">
+
+    <div class="wrapper">
+        <div class="table-cell">
+            <header id="site-header" role="banner">
+                <div class="container">
+                    [% INCLUDE 'header_logo.html' %]
+                    <a href="#main-nav" id="nav-link">Main Navigation</a>
+                </div>
+                [% INCLUDE 'header_nav.html' %]
```

Comment:
Yes, that would be pretty useful. Extracting the whole `<header>` element into its own template would be good, as I've had to override the whole thing in at least one cobrand.